### DISCUSSION TITLE
Change the width and height of po-modal

### DISCUSSION THREAD

**Discussion by Lourencobrah** [po-ui/po-angular#1302] [Angular] [answered]
Hello!

How can i change the dimensions of po-modal?

**Comment by rafaellmarques** [comment, marked as answer]
Boa tarde @Lourencobrah , tudo bem?
O PoModal possuí a propriedade `p-size` que recebe valores específicos para a largura do componente.

`p-size` - Define o tamanho da modal.

Valores válidos:
- sm (pequeno)
- md (médio)
- lg (grande)
- xl (extra grande)
- auto (automático)

> Quando informado auto a modal calculará automaticamente seu tamanho baseado em seu conteúdo. Caso não seja informado um valor, a modal terá o tamanho definido como **md**.

> Todas as opções de tamanho possuem uma largura máxima de **768px**.

Veja se isso lhe ajuda.
